mmv1/api/product.rb

```diff
@@ -46,6 +46,12 @@ class Product < Api::Object::Named
     # For example: `https://www.googleapis.com/compute/v1/`
     attr_reader :base_url
 
+    # A function reference designed for the rare case where you
+    # need to use retries in operation calls. Used for the service api
+    # as it enables itself (self referential) and can result in occasional
+    # failures on operation_get. see https://git.io/Jzu04
```

Comment:
Did GitHub shorten that link? I'd prefer we include the full link, since links can change under-the-hood in URL shorteners and it's more clear what you're clicking on in advance.

Reply:
I used git.io to shorten the url which is an official product, I can change it thought if you like.